requirements.pipx.txt

```diff
@@ -0,0 +1 @@
+awscli>=1.0,<2.0
```

Comment:
`awscli` is a tool.

According to the docs, tools are better installed via `pipx` (ala `npx`)